packages/sushi/src/router/liquidity-providers/UniswapV3Base.ts

```diff
@@ -435,25 +452,23 @@ export abstract class UniswapV3BaseProvider extends LiquidityProvider {
   getStaticPools(t1: Token, t2: Token): StaticPool[] {
     const currencyCombinations = getCurrencyCombinations(this.chainId, t1, t2)
 
-    const allCurrencyCombinationsWithAllFees: [
-      Type,
-      Type,
-      SushiSwapV3FeeAmount,
-    ][] = currencyCombinations.reduce<
-      [Currency, Currency, SushiSwapV3FeeAmount][]
-    >((list, [tokenA, tokenB]) => {
-      if (tokenA !== undefined && tokenB !== undefined) {
-        return list.concat([
-          [tokenA, tokenB, SushiSwapV3FeeAmount.LOWEST],
-          [tokenA, tokenB, SushiSwapV3FeeAmount.LOW],
-          [tokenA, tokenB, SushiSwapV3FeeAmount.MEDIUM],
-          [tokenA, tokenB, SushiSwapV3FeeAmount.HIGH],
-        ])
-      }
-      return []
-    }, [])
-
-    const filtered: [Token, Token, SushiSwapV3FeeAmount][] = []
+    const allCurrencyCombinationsWithAllFees: [Type, Type, number][] =
```

Comment:
@rouzwelt can you pls match the formatting so this whole block doesn't show as changed?

Reply:
this is the repo linter setup, it lints like this automatically, if I change it then the lint tests will fail